src/main/java/org/isf/medstockmovtype/rest/MedStockMovementTypeController.java

```diff
@@ -70,13 +70,13 @@
 	public ResponseEntity<List<MovementTypeDTO>> getMedicaldsrstockmovType() throws OHServiceException {
 		LOGGER.info("Retrieving all the movement types ...");
 		List<MovementType> movementTypes = manager.getMedicaldsrstockmovType();
-		List<MovementTypeDTO> mappedMvments = mapper.map2DTOList(movementTypes);
-		if(mappedMvments.isEmpty()) {
+		List<MovementTypeDTO> mappedMovments = mapper.map2DTOList(movementTypes);
+		if (mappedMovments.isEmpty()) {
```

Comment:
is the 'e' of movements missing intentionally?

Reply:
Nope; I'll fix it.

Reply:
Done.